aws/rudderstack/iam.tf

```diff
@@ -0,0 +1,45 @@
+resource "aws_iam_role" "rudderstack-role" {
+  name               = "rudderstack-s3-role"
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "arn:aws:iam::422074288268:root"
```

Comment:
We can use
data "aws_caller_identity" "current" {}
and ${data.aws_caller_identity.current.account_id}
for the AWS account

Reply:
You are right Andre, but this aws account is not ours, it's rudderstack's. They will assume that role to push to our S3 bucket the telemetry data.
https://www.rudderstack.com/docs/destinations/aws-iam-role-for-rudderstack/